src/libs/ReportUtils.ts

```diff
@@ -1275,58 +1368,57 @@ function getPersonalDetailsForAccountID(accountID) {
 
 /**
  * Get the displayName for a single report participant.
- *
- * @param {Number} accountID
- * @param {Boolean} [shouldUseShortForm]
- * @param {Boolean} shouldFallbackToHidden
- * @returns {String}
  */
-function getDisplayNameForParticipant(accountID, shouldUseShortForm = false, shouldFallbackToHidden = true) {
+function getDisplayNameForParticipant(accountID?: number, shouldUseShortForm = false, shouldFallbackToHidden = true): string | undefined {
     if (!accountID) {
         return '';
     }
 
     const personalDetails = getPersonalDetailsForAccountID(accountID);
+    // console.log(personalDetails);
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
     const formattedLogin = LocalePhoneNumber.formatPhoneNumber(personalDetails.login || '');
-
     // This is to check if account is an invite/optimistically created one
     // and prevent from falling back to 'Hidden', so a correct value is shown
-    // when searching for a new user while offline
-    if (lodashGet(personalDetails, 'isOptimisticPersonalDetail') === true) {
+    // when searching for a new user
+    if (personalDetails.isOptimisticPersonalDetail === true) {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
         return formattedLogin;
     }
+    const longName = personalDetails.displayName ? personalDetails.displayName : formattedLogin;
+
+    const shortName = personalDetails.firstName ? personalDetails.firstName : longName;
```

Comment:
@kubabutkiewicz I saw you tried the optional chaining first: `personalDetails.displayName ?? formattedLogin`. Was there any issue with using that approach? I suppose it showed the empty line if `personalDetails.displayName` was such?

Reply:
Is there any drawback to using just `const longName = personalDetails.displayName || formattedLogin`?

Reply:
`||` is not allowed on strings

Reply:
Correct in js, but will throw type errors in ts

Reply:
Thanks. I didn't know it was a TS-specific thing 👍

Reply:
I think I am not needed here any more 😄 Thanks @shubham1206agra